projects/vdk-plugins/vdk-ipython/tests/test_plugin.py

```diff
@@ -27,6 +30,14 @@ def ip(session_ip):
     session_ip.run_line_magic(magic_name="reset", line="-f")
 
 
+@contextmanager
+def get_vdk_ipython(session_ip, vdk_load_arguments_line=""):
+    session_ip.run_line_magic(magic_name="load_ext", line="vdk.plugin.ipython")
+    session_ip.run_line_magic(magic_name="reload_VDK", line=vdk_load_arguments_line)
+    yield session_ip
```

Comment:
Is this supposed to be a generator? I don't see the point of using yield if we'll just call it once.

Reply:
This is yield-based context manager. And it must be called exactly once. See https://docs.python.org/3/library/contextlib.html#contextlib.contextmanager 
 
 This allows me to do later this:
 ```
 with get_vdk_ipython() as ip:
     # code before yield is executed and ip is returned 
 # After going out of scope automatically clean up code after yield is executed. 
 ```